man/systemd.network.xml

```diff
@@ -4451,6 +4451,79 @@ Token=prefixstable:2002:da8:1::</programlisting></para>
     </variablelist>
   </refsect1>
 
+  <refsect1>
+    <title>[EarliestTxTimeFirst] Section Options</title>
+    <para>The [EarliestTxTimeFirst] section manages the queueing discipline (qdisc) of Earliest TxTime First
+    (etf).</para>
+
+    <variablelist class='network-directives'>
+      <xi:include href="tc.xml" xpointer="qdisc-parent" />
+      <xi:include href="tc.xml" xpointer="qdisc-handle" />
+
+      <varlistentry>
+        <term><varname>Clockid=</varname></term>
+        <listitem>
+          <para>The ETF (Earliest TxTime First) qdisc allows applications to control
+          the instant when a packet should be dequeued from the traffic control
+          layer into the netdevice.
+          Specifies the clock to be used by qdisc's internal timer for measuring
+          time and scheduling events. The qdisc expects that packets passing
+          through it to be using this same as the reference of their txtime timestamps.
+          It will drop packets coming from sockets that do not comply with that.
+          See tc-etf(8) for more details.</para>
+        </listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>Delta=</varname></term>
```

Comment:
we try to suffix settings names with the expected unit. DeltaNSec= hence, please